enterprise/views.py

```diff
@@ -380,6 +383,60 @@ def get_course_or_program_context(self, enterprise_customer, course_id=None, pro
             })
         return context_data
 
+    def get_page_language_context_data(
```

Comment:
i don't understand how these changes relate to the ticket?

Reply:
annoyingly with my additions the `get()` method failed quality tests because it was too long so I had to pull out some of the statements into another function